pages/common/colon.md

```diff
@@ -0,0 +1,9 @@
+# colon
+
+> Returns a successful exit status code of 0.
+> Use this with the operator `||` to make a command always exit with 0.
+> More information: <https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#colon>.
+
+- Return a successful exit code:
+
+`:`
```

Comment:
```suggestion
`:`

- Make a command always exit with 0:

`{{command}} || :`
```
@tricantivu in reality, the description mean using the operator after the commmand, and not before. IMO it would be better to move this to an example to make it more clear.

Reply:
My mistake, I misread the line. Although, I cannot see a scenario where a failed command should return exit code zero.

Reply:
It can be very useful if, for example, set -e is set and any non-zero status code would cause the script to exit. If there is a loop which searches for some files and it's ok if some file is not found because it can be created or something, this can be an useful trick.